data/hooks/diagnosis/10-ip.py

```diff
@@ -121,13 +131,12 @@ def ping(protocol, target):
     def can_resolve_dns(self):
         return os.system("dig +short ip.yunohost.org >/dev/null 2>/dev/null") == 0
 
-    def resolvconf_is_symlink(self):
-        return os.path.realpath("/etc/resolv.conf") == "/run/resolvconf/resolv.conf"
-
-    def resolvconf_points_to_localhost(self):
-        file_ = "/etc/resolv.conf"
-        resolvers = [r.split(" ")[1] for r in read_file(file_).split("\n") if r.startswith("nameserver")]
-        return resolvers == ["127.0.0.1"]
+    def good_resolvconf(self):
+        content = read_file(file_).strip().split("\n")
```

Comment:
Traceback (most recent call last):
  File "/usr/lib/moulinette/yunohost/diagnosis.py", line 168, in diagnosis_run
    code, report = hook_exec(path, args={"force": force}, env=None)
  File "/usr/lib/moulinette/yunohost/hook.py", line 335, in hook_exec
    returncode, returndata = _hook_exec_python(path, args, env, loggers)
  File "/usr/lib/moulinette/yunohost/hook.py", line 453, in _hook_exec_python
    ret = module.main(args, env, loggers)
  File "/usr/share/yunohost/hooks/diagnosis/10-ip.py", line 159, in main
    return IPDiagnoser(args, env, loggers).diagnose()
  File "/usr/lib/moulinette/yunohost/diagnosis.py", line 387, in diagnose
    items = list(self.run())
  File "/usr/share/yunohost/hooks/diagnosis/10-ip.py", line 44, in run
    good_resolvconf = self.good_resolvconf()
  File "/usr/share/yunohost/hooks/diagnosis/10-ip.py", line 135, in good_resolvconf
    content = read_file(file_).strip().split("\n")
NameError: global name 'file_' is not defined

Reply:
Damn yes sorry, forgot to actually fix this ... Fixed naow!